src/content/lib/utils/chrome-files.js

```diff
@@ -0,0 +1,149 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * ***** END LICENSE BLOCK *****
+ */
+
+
+const {NetUtil} = Cu.import("resource://gre/modules/NetUtil.jsm");
+const {httpRequest} = Cu.import("resource://gre/modules/Http.jsm");
+
+/**
+ * Utility object to handle chrome's file (i.e loaded from chrome://)
+ */
+export const ChromeFilesUtils = (function() {
+  let self = {
+   getChromeUrl: null,
+   readDirectory: null,
+   parseJSON: null,
+  };
```

Comment:
recently I've switched from `self`-style `FooBarUtils` modules to this style:

```js
export function getChromeUrl(path) {
  // ...
}
```

Importing changes from `import {ChromeFilesUtils} from "path/file";` to:

```js
import * as ChromeFilesUtils from "path/file";
```

The files itself I'm naming `foo-bar-utils.js` (including `-utils`).

I know it's just a minor change, but maybe you can apply it to this branch? Of course you may simply add a new commit to the current commit history.

Reply:
> The files itself I'm naming foo-bar-utils.js (including -utils).

Can't agree more, it's a lot easier when modules and source file have exactly the same name :smiley:. I made the changes in https://github.com/RequestPolicyContinued/requestpolicy/pull/881/commits/81a87774bbbee4e240684c947a57d3f8ddcd828c

> recently I've switched from self-style FooBarUtils modules to this style:
> ```javascript
> export function getChromeUrl(path) {
>  // ...
> }
> ```
>
> Importing changes from import {ChromeFilesUtils} from "path/file"; to:
> ```javascript
> import * as ChromeFilesUtils from "path/file";
> ```

I made the changes for `ChromeFilesUtils`, `l10n` and `I18nUtils` modules (see https://github.com/RequestPolicyContinued/requestpolicy/pull/881/commits/36b8a83191815782378985459b85c5719d21ed96).
But does the same apply also for `LocaleData` and `LocaleManager` ? Because in the first case I'm exporting a constructor (I guess nothing to change in the `export` part but maybe in the `import` part ?), and in the second case I'm exporting a singleton object.

Reply:
Thanks for the changes. No need to change `LocaleData`, because it's a class. I don't know about `LocaleManager` yet; I'll comment later / elsewhere.